### PR TITLE
Выбор билда клиента 

### PR DESCRIPTION
**Pull request by Scondic**
Introduced a MultiSelect component for handling multiple selections in forms. Integrated this component into the build client form specifically for selecting multiple platforms. This change also alters how some hooks function to accommodate the multi-platform selection.

### CODE REVIEW

File: src/features/notifications/ui/Notifications.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ export const Notifications = () => {
                   <span className="text-base font-semibold">{message}</span>
                 </div>
                 <span className="text-sm truncate h-10 text-wrap w-[calc(100%-24px)]">
-                  {details}
+                  {details ? details : "Детали отсутсвуют"}
```
Comment (Scondic) [inline src/features/notifications/ui/Notifications.tsx:77]: отсутствуют

File: src/widgets/notifications-hub/lib/useConnectionHub.ts [hunk 2]
```diff
@@ -31,13 +31,13 @@ export const useConnectionHub = () => {
 
         connection.on(
           "Notifications",
-          ({ message, details }: { message: string; details: string }) => {
+          ({ message, details }: { message: string; details?: string }) => {
```
Comment (Scondic) [inline src/widgets/notifications-hub/lib/useConnectionHub.ts:34]: Вынести в отдельный тип

```ts
type NotificationsParams { 
  message: string; 
  details: Nullable<string>;
}

type NotificationsParams { 
  message: string; 
  details: string | null;
}
```

File: src/widgets/notifications-hub/lib/useConnectionHub.ts [hunk 3]
```diff
@@ -31,13 +31,13 @@ export const useConnectionHub = () => {
...
             sonner(message, {
-              description: `${details.substring(0, 50)}...`,
+              description: details && `${details?.substring(0, 50)}...`,
```
Comment (Scondic) [inline src/widgets/notifications-hub/lib/useConnectionHub.ts:36]:
```ts
description: details && `${details.substring(0, 50)}...`,
```

File: src/widgets/notifications-hub/lib/useConnectionHub.ts [hunk 4]
```diff
@@ -31,13 +31,13 @@ export const useConnectionHub = () => {
...
               action: {
                 label: "Cкопировать",
```
Comment (Scondic) [inline src/widgets/notifications-hub/lib/useConnectionHub.ts:38]: Замени все "Скопировать" на "Копировать"

File: src/widgets/notifications-hub/lib/useConnectionHub.ts (hunk 2 above)
Comment (Scondic) [reply src/widgets/notifications-hub/lib/useConnectionHub.ts:34]: src/global.d.ts

```ts
type Nullable<BaseType> = BaseType | null;
```

Comment (Scondic) [reply src/widgets/notifications-hub/lib/useConnectionHub.ts:34]: src/nextjs.d.ts

**Comment by Arsenii1109** [comment]
@Scondic

**Pull request merged**